docs/Reference/CLI/Subcommands/Validator-Client.md

```diff
@@ -695,7 +790,64 @@ using Teku to sign blocks and attestations always uses its built-in slashing pro
     validators-external-signer-timeout: 2000
     ```
 
-Timeout in milliseconds for requests to the external signer. The default is 1000.
+Timeout in milliseconds for requests to the external signer. The default is 5000.
+
+### validators-external-signer-truststore
+
+=== "Syntax"
+
+    ```bash
+    teku vc --validators-external-signer-truststore=<FILE>
+    ```
+
+=== "Example"
+
+    ```bash
+    teku vc --validators-external-signer-truststore=websigner_truststore.p12
+    ```
+
+=== "Environment variable"
+
+    ```bash
+    TEKU_VALIDATORS_EXTERNAL_TRUSTSTORE=websigner_truststore.p12
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    validators-external-signer-truststore: "websigner_truststore.p12"
+    ```
+
+PKCS12 or JKS keystore used to trust external signer's self-signed certificate or CA certificate
+which signs the external signer's certificate.
+
+### validators-external-signer-truststore-password-file
+
+=== "Syntax"
+
+    ```bash
+    teku vc --validators-external-signer-truststore-password-file=<FILE>
+    ```
+
+=== "Example"
+
+    ```bash
+    teku vc --validators-external-signer-truststore-password-file=truststore_pass.txt
+    ```
+
+=== "Environment variable"
+
+    ```bash
+    TEKU_VALIDATORS_EXTERNAL_TRUSTSTORE_PASSWORD_FILE=truststore_pass.txt
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    validators-external-signer-truststore-password-file: "truststore_pass.txt"
+    ```
+
+Password file used to decrypt the keystore.
```

Comment:
`Password file to decrypt keystore (PKCS12/JKS) that will be used to trust external signer's self-signed certificate`
that's whats in our usage information, the context for why the keystore is used is probably important but could verify if its missing anything with @usmansaleem